src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -88,8 +88,8 @@ export function ModelingSidebar({ paneOpacity }: ModelingSidebarProps) {
     },
     {
       id: 'share-link',
-      title: 'Create share link',
-      sidebarName: 'Create share link',
+      title: 'Share part via Zoo link',
+      sidebarName: 'Share part via Zoo link',
```

Comment:
@franknoirot Trying to see if this would be better? No bracket like before. I used _part_ and not _model_ to be consistent with the other ones (Make and Export) but I think we'll want to use _model_ once we can share assemblies.

Potentially we might want to disable this in the presnce of import statements too?

Reply:
I like this because it puts the important word first too. Disabling is a solid idea, we can pass a reason into those buttons for why they're disabled too.

Reply:
Cool! Makes sense